bats/tests/helpers/paths.bash

```diff
@@ -1,17 +1,64 @@
+# PATH_BATS_ROOT and PATH_BATS_HELPERS are already set by load.bash
```

Comment:
Rather than a comment, we can just assert that they _are_ set (this avoids issues with incorrect load orders).

Reply:
I'm not worried about load order because `load.bash` is the function that sources everything else. It is just that I expect **all** `PATH_*` variables to be defined in `paths.bash`, so I wanted to document the 2 exceptions. I can remove the comment, if you think it is not useful, but I don't see any value in adding an assertion.